tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -32,6 +32,12 @@ build --copt=-mavx --host_copt=-mavx
 # See https://docs.bazel.build/versions/main/skylark/performance.html#performance-profiling
 build --profile=/tf/pkg/profile.json
 
+# Store the execution log binary in the mounted artifact directory.
+# This log is recommended to debug missing cache hit on the same machine and on different machines.
+# See more at step 2 and 3c in:
+# https://docs.bazel.build/versions/main/remote-caching-debug.html
+build --execution_log_binary_file=/tf/pkg/exec_cuda.log
```

Comment:
Can you change this to exec_gpu to match the name of the config?

Reply:
Done. It was just for having a more future proof naming on GPU devices (RCOM, Intel new discrete GPU etc..)